entrypoint.sh

```diff
@@ -4,6 +4,9 @@ if [ $INPUT_DRY_RUN ]; then INPUT_DRY_RUN='--dry-run'; else INPUT_DRY_RUN=''; fi
 if [ $INPUT_CHANGELOG ]; then INPUT_CHANGELOG='--changelog'; else INPUT_CHANGELOG=''; fi
 if [ $INPUT_PRERELEASE ]; then INPUT_PRERELEASE="--prerelease $INPUT_PRERELEASE"; else INPUT_PRERELEASE=''; fi
 if [ "$INPUT_COMMIT" == 'false' ]; then INPUT_COMMIT='--files-only'; else INPUT_COMMIT=''; fi
+if [ "$INPUT_COMMITIZEN_VERSION" == 'latest' ]; then COMMITIZEN_VERSION=''; else COMMITIZEN_VERSION="commitizen==$INPUT_COMMITIZEN_VERSION"; fi
```

Comment:
Shouldn't the case `latest` define `COMMITIZEN_VERSION="commitizen"` ?
Else when we install requirements we have `pip install ''` - I think that's why tests are failing

```suggestion
if [ "$INPUT_COMMITIZEN_VERSION" == 'latest' ]; then COMMITIZEN_VERSION="commitizen"; else COMMITIZEN_VERSION="commitizen==$INPUT_COMMITIZEN_VERSION"; fi
```